rest-service/manager_rest/resource_manager.py

```diff
@@ -45,6 +45,7 @@
     update_inter_deployment_dependencies,
     verify_blueprint_uploaded_state,
     compute_rule_from_scheduling_params,
+    get_labels_list
```

Comment:
Trailing comma please, so that the next person who has to add something to (or delete something from) this list only has to do a one line change too!